### DISCUSSION TITLE
Paypal Donation

### DISCUSSION THREAD

**Discussion by elcotek** [oobabooga/text-generation-webui#1236] [General]
I would like to donate to the project. I would like to donate via Paypal but I can't find a donation link.

Could oobabooga possibly provide a donation link for PayPal?

Thank you for answering my question in advance.

**Comment by bmoconno** [comment]
In the upper right of this GitHub repo there is a "Sponsor" button which will take you to the [ko-fi link]([ko-fi.com/oobabooga](https://ko-fi.com/oobabooga)) for @oobabooga, you can use PayPal as your payment method on ko-fi.

**Comment by elcotek** [comment]
sorry, but i can't see any Payment selection on his payment page. ->
![grafik](https://user-images.githubusercontent.com/18084281/232233444-b54dbe7c-5f23-406c-bd27-01c2a82abfd7.png)

**Reply by bmoconno** [reply]
Ohh man, yeah I was reading their site wrong. You can receive money via PayPal but not send it, it seems. Sorry about that confusion. I've donated $10 [in your name](https://Ko-fi.com/home/coffeeshop?txid=5423bd24-cd53-4473-b778-08c4bba7e016&mode=public&img=ogiboughtsomeone). 😄
